### PR TITLE
Storybook/grid

### PR DESCRIPTION
**Pull request by beatrizbeato**
# Description
re-structure Grid components (customised from react-awesome-styled-grid) on storybook for easier integration

### Integration
to create grid layout  use custom components:  Grid, GridRow and GridCol  from Grid.tsx  on storybook
check layout use cases   for the Melon app at Grid.stories.tsx

### Disclaimers
after implementation, might need a review on paddings/sizing according to data/copy fitting.

### CODE REVIEW

File: src/storybook/components/Grid/Grid.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,91 @@
...
+    </Grid>
+  );
+};
 
-export const Fund: React.FC = () => <Grid layout="fund" />;
+export const TwoClumns: React.FC = () => {
+  return (
+    <Grid>
+      <GridRow>
+        <GridCol xs={12} sm={6} >
```
Comment (fubhy) [inline src/storybook/components/Grid/Grid.stories.tsx]: Please make sure that the formatting of the code adheres to our standards. There are a lot of white space errors in various places. E.g. a whitespace b efore a closing html tag, etc.

File: src/storybook/components/Grid/Grid.stories.tsx [hunk 2]
```diff
@@ -1,12 +1,91 @@
...
+          <Block>Two columns, stacked on xs-screens <br></br><hr></hr>xs={12} sm={6}</Block>
+        </GridCol>
+        <GridCol xs={12} sm={6} >
+          <Block> Two columns, stacked on xs-screens <br></br><hr></hr>xs={12} sm={6}</Block>
```
Comment (fubhy) [inline src/storybook/components/Grid/Grid.stories.tsx]: Also here there should for instance not be a whiotespace ```<Block> Two```

File: src/storybook/components/Grid/Grid.stories.tsx [hunk 3]
```diff
@@ -1,12 +1,91 @@
...
+      </GridRow>
+    </Grid>
+  );
+};
 
-export const CenteredSmall: React.FC = () => <Grid layout="small-center" />;
+export const StandaloneSmall: React.FC = () => {
+  return (
+    <Grid>
+      <GridRow  justify="center">
```
Comment (fubhy) [inline src/storybook/components/Grid/Grid.stories.tsx]: Two whitspace ```<GridRow  justify```

File: src/storybook/components/Grid/Grid.stories.tsx [hunk 4]
```diff
@@ -1,12 +1,91 @@
...
+        <GridCol xs={12} sm={6} md={4} lg={4} >
```
Comment (fubhy) [inline src/storybook/components/Grid/Grid.stories.tsx]: Whitespace before end of tag ```lg={4} >```

File: src/storybook/components/Grid/Grid.stories.tsx [hunk 5]
```diff
@@ -1,12 +1,91 @@
...
+          <Block> Stand-alone form with small fields <hr></hr> xs={12} sm={8} md={5} lg={4} </Block>
```
Comment (fubhy) [inline src/storybook/components/Grid/Grid.stories.tsx]: This description (`xs={12} sm={6} md={4} lg={4}`) does not match the GridCol above.

File: src/storybook/components/Grid/Grid.stories.tsx (hunk 1 above)
Comment (beatrizbeato) [reply src/storybook/components/Grid/Grid.stories.tsx]: of course , will be more thorough on the next PR 👍

**Comment by fubhy** [comment]
Thanks! Merged.

**Pull request merged**